library/js-dev-b2b-intro-to-html-css-fundamentals-XX-assessment/quizkey.md

```diff
@@ -0,0 +1,235 @@
+
+   ```html
+   <main>
+     <h7>Welcome to Thinkful</h7>
+     <p>This is a simple paragraph created within HTML.</p>
+     <div></div>
+   </main>
+   ```
+
+   A. True
+
+   B. False
+
+   **Answer:** B. False
+
+   **Distractor Rationale: However, HTML won't warn you with any visible errors if you use a heading tag above <h6>, so it's up to you to keep an eye out.**
+
+9. How many different paragraph sizes are possible in HTML? In other words, `<p1>` through what number?
+
+A. Unlike heading elements, paragraph elements don't have sizes.
+
+B. `<p6>`
+
+C. `<p7>`
+
+D. `<p8>`
+
+**Answer:** A.
+
+**Distractor Rationale: Do paragraph elements have sizes associated with them?**
+
+10. You'd like to display an image on a web page and set its width to `600px`. Which of the following options are correct?
+
+A. `<img src="images/coder.jpg" width="600"></img>`
+
+B. `<img src="images/coder.jpg" width="600"></img>`
+
+C. `<img src="images/coder.jpg" width="600" />`
+
+D. `<img width="600" src="images/coder.jpg" />`
+
+E. `<img width="600" source="images/coder.jpg" />`
+
+F. `<img src="images/coder.jpg" alt="600" />`
+
+**Answers:** C and D.
+
+**Distractor Rationale: Keep in mind `img` tags are self-closing, and the order of the attributes (`src` and `width`) doesn't matter.**
+
+11. You're creating a web page that renders a few paragraphs:
+
+Heading: `Coffee or Tea?`
+Subheading: `What do you prefer?`
+Content: `The world is made up of two types of people: Coffee drinkers and tea drinkers.`
+
+Which of the following approaches is most preferable, and why?
+
+Approach 1:
+
+```html
+<main>
+  <p>Coffee or Tea?</p>
+  <p>What do you prefer?</p>
+  <p>
+    The world is made up of two types of people: Coffee drinkers and tea
+    drinkers.
+  </p>
+</main>
+```
+
+Approach 2:
+
+```html
+<main>
+  <h1>Coffee or Tea?</h1>
+  <h2>What do you prefer?</h2>
+  <p>
+    The world is made up of two types of people: Coffee drinkers and tea
+    drinkers.
+  </p>
+</main>
+```
+
+Approach 3:
+
+```html
+<main>
+  <p>
+    Coffee or Tea? What do you prefer? The world is made up of two types of
+    people: Coffee drinkers and tea drinkers.
+  </p>
+</main>
+```
+
+A. Approach 1 is preferred because the use of `<p>`/`</p>` tags separates each paragraph from the one above it and the one below it. If all the tags are removed, the blocks of text will simply lump together and no longer be separated into discrete paragraphs.
+
+B. Approach 2 is preferred because it uses the appropriate HTML elements to break up, organize, and highlight content in the web page.
+
+C. Approach 3 is preferred because it uses only one set of `<p>` tags to display the paragraphs, thereby reducing code repetition. Since it uses fewer unnecessary HTML elements, the code is cleaner.
+
+D. None of the above.
+
+**Answer:** B. Approach 2 is preferred.
+
+**Distractor Rationale: Which approach best organizes the content using the appropriate HTML elements?**
```

Comment:
@kchia do you think you could come up with one more question so that we could have an even multiple of 4? When we have a MC assessment like this the passing threshold will be 75%, so it will be nice for students to know they have to get 9/12 to pass.